components/ResultsPageComponents/SharedComponents/ContentStyles.jsx

```diff
@@ -0,0 +1,17 @@
+import { Box } from "@chakra-ui/react";
+
+const ResultContentSection = (props) => {
+  const { isShaded, children } = props;
+  return (
+    <Box
+      width="100vw"
+      py="inherit"
+      px="inherit"
+      bg={isShaded ? "#FAFAFA" : "inherit"}
+    >
+      {children}
+    </Box>
+  );
+};
+
+export { ResultContentSection };
```

Comment:
```suggestion
}

```

Reply:
I was going with the intention that I will probably have more than one content style to configure. And in trying to not have too many .jsx files in the codebase, I was going to have them all in the same file. Either way, if you prefer to have one component to one file, I can facilitate that.